contrib/babelfishpg_tsql/src/pl_handler.c

```diff
@@ -1817,6 +1834,10 @@ pltsql_sequence_datatype_map(ParseState *pstate,
 		 * Identity column drops the typmod upon sequence creation
 		 * so it gets its own check
 		 */
+
+		if (typmod_p == -1)
```

Comment:
please add an appropriate comment for this change.
1179652 is for Numeric(18,0), is that expected behaviour?

Reply:
In case of UDT typmod_p is equal to -1, but for a create sequence query for_identity is false, therefore the code enters the if condition below and the precision and scale is then calculated using typmod_p, which are wrong since typmod==-1. Hence, I've added this condition. I'll add the relevant comment.